plugins/dns/custom/README.md

```diff
@@ -0,0 +1,35 @@
+# Configuration
+
+This plugin provides a DNS integration for OpenShift Enterprise version 2.2.
+The plugin essentially allows calling of a local script that can be used to integrate with a remote system as required.  In the example the script calls local nsupdate commands.
+
+The configuration file for the plugin is ```/etc/openshift/plugins.d/openshift-origin-dns-custom.conf```
+
+Three variables in the configuration file define the location of the update server:
```

Comment:
This mentions three variables in the configuration file. There looks to only be one and it doesn't define the location of the update server.